components/infobox/wikis/apexlegends/infobox_person_player_custom.lua

```diff
@@ -189,6 +190,8 @@ function CustomPlayer:defineCustomPageVariables(args)
 	else
 		Variables.varDefine('isplayer', 'true')
 	end
+	local lowercaseInput = _args.input and _args.input:lower() or nil
+	Variables.varDefine('input', _INPUTS[lowercaseInput] or _INPUTS['default'])
```

Comment:
Unless there's a need to store `input` as a wiki variable because it needs to be accessed by SMW, please make it a function instead